scripts/perftune.py

```diff
@@ -606,6 +606,18 @@ def tune(self):
         # did not receive an acknowledgment from connecting client.
         fwriteln_and_log('/proc/sys/net/ipv4/tcp_max_syn_backlog', '4096')
 
+        self.tune_tcp_mem()
+
+    def tune_tcp_mem(self):
+        page_size = os.sysconf("SC_PAGE_SIZE")
+        phys_mem = os.sysconf("SC_PAGE_SIZE") * page_size
```

Comment:
This is a strange construct: you are effectively making `phys_mem == os.sysconf("SC_PAGE_SIZE")^2`.

I think you meant to multiply the PAGE_SIZE by NUM_PAGES.

However I don't understand why even this is needed since `tcp_mem` is configured in PAGEes anyway.

Reply:
One way to do that is using `psutil.virtual_memory().total` (total memory in bytes):

```
phys_mem = psutil.virtual_memory().total`
```

And our currently shipped relocatable Python3.11 ships psutil.

The rest of your code would remain the same.

Reply:
A nitpick:
If you decide to use `psutil` as above I'd also recommend changing the variable name to something else because otherwise a construct "phys_mem = ... virtual_mem.total` is confusing.

Something like "total_mem" would be less  contradictory IMO.